File: TouchPOS/Models/SysConfigModels.vb


Imports MD
Public Class SysConfig
    Public Class value
        Public Respon As New ResponseModels
        Public Data As List(Of SysConfigModels)

    End Class
    Public Class SysConfigModels
        Public Property ITEMSNO As String = ""
        Public Property CUSGROUP As String = ""
        Public Property VENDOR_GETWAY As String = ""
        Public Property CONN_LINK As String = ""

        Public Property WALLET_TYPE As String = ""
        Public Property USESTOCK As String = ""
        Public Property POSTYPE As String = ""
    End Class
End Class

File: TouchPOS_API/Models/SysConfigModels.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace TouchPOS_API.Models
{
    public class SysConfigModels
    {
        public string ITEMSNO { get; set; }
        public string CUSGROUP { get; set; }
        public string VENDOR_GETWAY { get; set; }
        public string CONN_LINK { get; set; }
        public string WALLET_TYPE { get; set; }
        public bool USESTOCK { get; set; }

        public string POSTYPE { get; set; }
        
    }
}